SearchValueControllerTest.cls

```diff
@@ -33,6 +33,8 @@ private class SearchValueControllerTest {
         incontroller.searchString = 'test';
         incontroller.explore();
         incontroller.searchString.length();
```

Comment:
@pranalimm This is not necessary remove it. Because in next line you have already used this method.